torch/_inductor/utils.py

```diff
@@ -3245,12 +3245,12 @@ def tabulate_2d(elements: Sequence[Sequence[T]], headers: Sequence[T]) -> str:
         for i, e in enumerate(row):
             widths[i] = max(widths[i], len(str(e)))
     lines = []
-    lines.append("|".join(f" {h:{w}} " for h, w in zip(headers, widths)))
+    lines.append("|".join(f" {str(h).ljust(w)} " for h, w in zip(headers, widths)))
```

Comment:
False positive for `SET_LINTER`, which proposes the following fix:

```diff
- "|".join(f" {h:{w}} "
+ "|".join(f" {h:OrderedSet([w])} "
```

Here `{w}` is not a set. It is interpreted as a format spec in f-string. cc @rec

Reply:
I'll file an issue against myself, but I'm out until Monday! 🙂 

You can do `# noqa: set_linter` on the line for the moment.